### PR TITLE
JESI-2870

### PR DESCRIPTION
**Pull request by axrs**
[JESI-2870](https://jesims.atlassian.net/browse/JESI-2870)

https://github.com/jesims/backpack/pull/46
https://github.com/jesims/jesi-web/pull/700
https://github.com/jesims/jesi/pull/932

### CODE REVIEW

**Review by axrs — COMMENTED** [review]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,20 @@
+## 0.2.0
+==================
+
+Added:
+* Anchors are now broken into `anchor` and `anchor-query`
```
Comment (BadAlgorithm) [inline CHANGELOG.md:4]: Add tests to cover the scenario with an anchor with an empty query

**Review by BadAlgorithm — CHANGES_REQUESTED** [review]

**Review by BadAlgorithm — APPROVED** [review]

**Review by AndreTheHunter — APPROVED** [review]
